Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5855,6 +5856,78 @@ def com_google_fonts_check_metadata_can_render_samples(ttFont, family_metadata):
        yield PASS, "OK."
 
 
+def count_islands(imagefont, character):
+    from PIL import Image, ImageDraw
```

Comment:
I don't know how PIL draws text. My idea was to use FreeType with hinting enabled. If PIL uses FT with hinting, ignore this comment. These accent collisions seem to be quite common on Win only, hence why I want hints.